Security/src/Test-ProxyLogon.ps1

```diff
@@ -220,7 +223,11 @@ begin {
                 }
 
                 function Get-AgeInDays {
-                    param ( $dateString )
+                    param (
+                        [Parameter( ValueFromPipeline = $true )]
+                        $dateString
+                    )
```

Comment:
If you add acceptance from pipeline, make sure to put the main logic into a process block.
It may work, as we only offer one item each time, but should be considered good practice anyway.

If you don't add any of begin/process/end blocks, all your code executes in end.

Reply:
That I do know. I agree and prefer a more robust syntax (even if verbose) anyhow.  It works in this case because we only send a single value that then persists through to `End{}` See other comment, I'm not sure it's worth it.

Reply:
That's actually causing a build failure:

![image](https://user-images.githubusercontent.com/4518572/110805734-fd10f280-8246-11eb-9993-47d65a6f5359.png)

Hey Steven, please run `.build\CodeFormatter.ps1 -Save`. That will fix the formatting and save the changes automatically, and it will report any PSScriptAnalyzer issues that need to be corrected manually, such as the process block thing.

Once CodeFormatter comes back clean, if you push to this branch again the build should succeed. Thanks!

Reply:
Understood. I'm sorry I'm having trouble running CodeFormatter due to environment security.  We can abandon this pull request for now, and I'll refile as soon as I can. Sorry for the trouble.